.github/workflows/build_android.yml

```diff
@@ -16,7 +16,7 @@ env:
 
 jobs:
   build:
-    runs-on: macos-latest
+    runs-on: macos-12
```

Comment:
Need to explicitly use macos as the latest mac has a new version of Java that doesn't work with android SDK sdkmanager.